### ISSUE TITLE
[Featute request] Bring the old tabs behaviour back 

### ISSUE THREAD

**Issue by fabioromeo** [TwidereProject/TwidereX-Android#541]
The original Twidere had the best tab behaviour I never saw in any app: it allowed me to put the timeline from each account on its tab, and all I had to do to see each account was to swipe between those tabs. That made browsing incredibly convenient, and I could even set a single tab to receive notifications from all accounts. It was amazing. 

Please, bring that back.

**Comment by nevillepark** [comment]
You could even, if you wanted, combine *all* your home timelines like with notifications.

**Comment by msravi** [comment]
Yes. It is super-convenient to have different lists and swipe between the timelines of those lists. Please bring this back. Even twitter's own mobile web interface seems to have recognized how good it was, and moved to such an interface (even though it isn't as configurable as what the original twidere has)

**Comment by fabioromeo** [comment]
> You could even, if you wanted, combine _all_ your home timelines like with notifications.

Oh, totally! The beauty of this feature was its flexibility, I hope it does comes back.
